data/lib/debugging/dump.lua

```diff
@@ -3,7 +3,7 @@ function dumpLevel(input, level)
 	local indent = ''
 
 	for i = 1, level do
-		indent = indent .. '    '
+		indent = indent .. '	'
```

Comment:
In this case it is better to use the escape value of the TAB instead of adding an real tab
```lua
indent = indent .. '\t'
table.insert(lines, indent .. '\t[' .. k .. '] = ' .. dumpLevel(v, level + 1))
```